pkg/phlaredb/phlaredb.go

```diff
@@ -50,7 +50,7 @@ type ParquetConfig struct {
 
 func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.StringVar(&cfg.DataPath, "pyroscopedb.data-path", "./data", "Directory used for local storage.")
-	f.DurationVar(&cfg.MaxBlockDuration, "pyroscopedb.max-block-duration", 3*time.Hour, "Upper limit to the duration of a Pyroscope block.")
+	f.DurationVar(&cfg.MaxBlockDuration, "pyroscopedb.max-block-duration", 1*time.Hour, "Upper limit to the duration of a Pyroscope block.")
```

Comment:
👍🏻 

Just thinking out-loud: now that we have compactor, smaller source blocks are preferable. I'm also wondering how it relates to the ingestion window. I suppose those should be "aligned", e.g the current default window duration is one hour, therefore 1h max block duration fits it nicely (so that at most two blocks of the same time range co-exist)

Reply:
Yes a validation for this would be nice, or at least a warning when the windows is totally out of boudns (e.g. more than X head blocks required)